management-app/src/main/java/com/nearsoft/fgaribay/mgmt/ManagementApplication.java

```diff
@@ -0,0 +1,19 @@
+package com.nearsoft.fgaribay.mgmt;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+@SpringBootApplication
+public class ManagementApplication extends SpringBootServletInitializer {
```

Comment:
`SpringBootServletInitializer` is a feature added in Spring Boot 2.0, which is not the version required for this project.